app/assets/stylesheets/utilities/_background.scss

```diff
@@ -1,10 +1,6 @@
 .bg-gray-lighter { background-color: $gray-lighter; }
 .bg-light-blue { background-color: $blue-light; }
 .bg-lightest-blue { background-color: $blue-lightest; }
-// scss-lint:disable ImportantRule
-.bg-white { background-color: $white !important; }
-// scss-lint:enable ImportantRule
```

Comment:
This was originally introduced in #4647 for the read-only SSN field ([related comment](https://github.com/18F/identity-idp/pull/4647#discussion_r571043229)). The field was removed as part of #5265, so it's no longer needed, and we should want to avoid `!important`.